content/docs/cdn-links.md

```diff
@@ -1,37 +1,37 @@
 ---
 id: cdn-links
-title: CDN Links
+title: CDN लिङ्कहरू
 permalink: docs/cdn-links.html
 prev: create-a-new-react-app.html
 next: hello-world.html
 ---
 
-Both React and ReactDOM are available over a CDN.
+दुबै React र ReactDOM हरु CDN मा उपलब्ध छन्।
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 ```
 
-The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
+माथिको संस्करणहरू केवल डेभलपमेन्टका लागि मात्र हुन्, र प्रोडक्सनको लागि उपयुक्त छैन। React को संकुचित र अनुकूलित बनाईएको संस्करणहरू यहाँ उपलब्ध छन्:
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 ```
 
-To load a specific version of `react` and `react-dom`, replace `16` with the version number.
+`react` र `react-dom` को विशेष संस्करण प्रयोग गर्नको लागि, `16` लाई संस्करण अंकले बदल्नुहोस।
 
-### Why the `crossorigin` Attribute? {#why-the-crossorigin-attribute}
```

Comment:
I don't know if we should translate `Attribute` because it has some meaning attached to it?

Reply:
Agreed. Keeping `Attribute` as it is makes more sense to me too.
I'll update the PR.